ew_agents/model_loader.py

```
@@ -0,0 +1,112 @@
+                self.model_name,
+                trust_remote_code=True,
+                use_fast=False
+            )
+            
+            # Load model with quantization for efficiency
+            self.model = AutoModelForCausalLM.from_pretrained(
+                self.model_name,
+                torch_dtype=torch.float16,
+                device_map="auto",
+                trust_remote_code=True,
+                load_in_8bit=True if self.device == "cuda" else False
```

Ruff (0.12.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)